cms/service/EvaluationService.py

```diff
@@ -921,7 +925,11 @@ def invalidate_submission(self,
                     submission_id,
                     dataset_id} == {None}
                 else None,
-                participation_id, task_id, submission_id, dataset_id, session)
+                participation_id,
+                # Provide the task_id only if the entire task has to be
+                # reevaluated and not only a specific dataset.
```

Comment:
just out of curiosity, is it even possible to reevaluate a task and not just a dataset?

Reply:
AFAIK AWS has no such option, but the RPC API allows it.